features/gezag-minderjarige/4b.1-ouder-of-partner-overleden-of-onbevoegd-tot-gezag.feature

```diff
@@ -0,0 +1,457 @@
+#language: nl
+
+Functionaliteit: 4b.1 - Ouder of partner overleden of onbevoegd tot gezag
+  Precondities voor deze vraag:
+  | 1.1 Staat persoon (minderjarige) als ingezetene in de BRP?             | Ja                                                                       |
+  | 1.2 Is persoon a. minderjarig en b. niet overleden?                    | Ja                                                                       |
+  | 1.3 Is minderjarige naar het buitenland geëmigreerd geweest?           | Nee                                                                      |
+  | 1.3a Is minderjarige in buitenland geboren?                            | Nee of geadopteerd met NL akte                                           |
+  | 1.4 Is uitspraak gezag aanwezig                                        | Nee                                                                      |
+  | 2.1 Hoeveel juridische ouders heeft minderjarigen                      | Een_ouder                                                                |
+  | 2b.1 is staande huwelijk of partnerschap geboren                       | Ja                                                                      |
+
+  Onbevoegd tot het gezag zijn minderjarigen, zij die onder curatele zijn gesteld en zij wier geestvermogens zodanig zijn gestoord, om het
+  gezag uit te kunnen oefenen (1:246 BW). Daarnaast zijn overleden ouders ook onbevoegd tot het gezag.
+
+  Mogelijke uitkomsten:
+  1) Nee
+  -> GG: Ouder en echtgenoot/partner hebben gezamenlijk het gezag (1:253sa BW)
+  a. meemoeders: kind tijdens huwelijk/gp geboren, niet erkend en er is geen sprake van een verklaring onbekende donor
+  b. kind vóór 1 april 2014 geboren tijdens geregistreerd partnerschap van paren verschillend geslacht, kind is niet erkend (Zie 2b.1 onder 1)
+  2) Ja, de niet-ouder
+  -> OG1: De ouder heeft alleen het gezag (1:253f BW)
+  3) Ja, de ouder
+  -> V: De niet-ouder heeft alleen het gezag (1:253x BW).
+  4) Ja, beide ouders
+  -> G: Er is (tijdelijk) geen gezag (1:253g lid 1 BW
+
+  Op het moment dat de moeder weer bevoegd wordt tot het gezag, verkrijgt zij van rechtswege het gezag, tenzij op dat tijdstip een ander met het
+  gezag is belast (art. 1:253b lid 2 BW). Dit geldt ook o.g.v. art. 1:253q lid 1 BW.
+  Let op: De kinderrechter kan een minderjarige moeder o.b.v. artikel 1:253ha lid 1 BW meerderjarig verklaren. Dit is nu niet in de BRP te achterhalen,
+  dit is wel zichtbaar als aantekening in het Gezagsregister.
+
+  Hoe achterhalen:
+  Niet overleden: op de persoonslijst van de ouder(s) mag de datum opschorting bijhouding niet voorkomen.
+  Meerderjarig: de geboortedatum van de ouder(s) moet 18 jaar of hoger zijn.
+  Niet onder curatele: op de persoonslijst van de ouder(s) mag de indicatie curatele niet voorkomen.
```

Comment:
helemaal niet of alleen in de actuele categorie?

Reply:
verduidelijkt dat dit opgenomen is in de gezagsverhouding